spec/requests/publishers/statistics_spec.rb

```diff
@@ -0,0 +1,98 @@
+require "rails_helper"
+
+RSpec.describe "Publisher statistics" do
```

Comment:
```suggestion
RSpec.describe "Publisher all time statistics" do
```

And maybe we could rename the file to `all_time_statistics_spec.rb` too?